services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+from typing import Optional, List
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None
+               ) -> Movie:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids)
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
```

Comment:
```suggestion
    if genres_ids:
        queryset = queryset.filter(genres__id__in=genres_ids)
    if actors_ids:
        queryset = queryset.filter(actors__id__in=actors_ids)
```

Reply:
@EdAlekseiev Sorry, I don't quite understand where the code needs to be corrected